caffe2/onnx/backend.cc

```diff
@@ -515,6 +515,59 @@ Caffe2Ops Caffe2Backend::CreateConvPoolOpBase(
   return CommonOnnxNodeToCaffe2Ops(onnx_node, opset_version);
 }
 
+Caffe2Ops Caffe2Backend::CreatePadPool(OnnxNode* onnx_node, int opset_version) {
+  auto& node = onnx_node->node;
+  auto& attributes = onnx_node->attributes;
+  Caffe2Ops ret;
+  // Pad
+  bool padding = false;
+  auto pad_a = dummy_->NewDummyName();
+  if (attributes.HasAttribute("count_include_pad")) {
+    auto count_include_pad = attributes.get<int64_t>("count_include_pad", 0L);
+    if (count_include_pad == 1) {
```

Comment:
You can enhance the check, if all pads are 0, no need to insert PadImage operator.